infra/build/functions/build_and_run_coverage.py

```diff
@@ -295,11 +295,13 @@ def get_fuzz_introspector_steps(  # pylint: disable=too-many-locals, too-many-ar
   coverage_url = (f'{build_lib.GCS_URL_BASENAME}{bucket_name}/{project.name}'
                   f'/reports/{coverage_report_latest}/linux')
 
-  build_steps.extend(
-      build_lib.download_coverage_data_steps(project.name,
-                                             coverage_report_latest,
-                                             bucket_name, build.out,
-                                             config.testing))
+  download_coverage_steps = build_lib.download_coverage_data_steps(
+      project.name, coverage_report_latest, bucket_name, build.out,
+      config.testing)
+  if not download_coverage_steps:
+    logging.info('Skipping introspector build for %s.', project.name)
```

Comment:
We should be more explicit here about why we're skipping this. i.e. no coverage data found. Also use `logging.warning` instead here.

Reply:
Applied your comment.